### DISCUSSION TITLE
New version without Zoom Client

### DISCUSSION THREAD

**Discussion by wifilogo** [sircharlo/meeting-media-manager#2700] [Q&A] [answered]
Hello.
Congratulations on your good work!
Will the Zoom Client not be available anymore in the new version?
Thanks!

**Comment by e-Feitosa** [comment, marked as answer]
> Olá. Parabéns pelo seu bom trabalho! O Zoom Client não estará mais disponível na nova versão? Obrigado!

**This option (Zoom Client) will be discontinued**

![image](https://github.com/user-attachments/assets/dcc958ac-444b-41a8-a64c-4d4cecee5eec)

**Reply by sircharlo** [reply]
Correct, the new version does not include the Zoom integration for the moment.
